### DISCUSSION TITLE
How to check Issue/PR went into which release

### DISCUSSION THREAD

**Discussion by rahul-paigavan** [istio/istio#50180] [General]
Hi,

I want to know in which release a specific issue or a PR went into.
I tried going through the release notes and it is very difficult to find the issue PR over various releases.

Is there an easy way to find out this thing?

**Comment by keithmattix** [comment]
If the PR was merged, scroll to the bottom of the PR and check the commit that was merged into master. If you click that commit, it will show you what branches/tags that commit is present in. In this example, the commit in question is only on master, to be added in a future release. The 3rd image shows what it will look like if the commit is already in a tag/release
<img width="888" alt="image" src="https://github.com/istio/istio/assets/1531662/2edca536-f7d7-4b5d-92c0-c28767145604">
<img width="391" alt="image" src="https://github.com/istio/istio/assets/1531662/54f69717-308f-4c84-b270-e9b60841d8c4">
<img width="280" alt="image" src="https://github.com/istio/istio/assets/1531662/408f0ab3-8908-4e57-a643-36aecde01683">

**Reply by rahul-paigavan** [reply]
Wow, great.
Thanks for quickly answering this.

**Comment by ericvn** [comment]
Note that there are also cherry-picks for a PR which may be used to create a patch in a prior release-branch. Similar logic to the above to see if a PR might have a tag to show it was actually pulled into a patch build.

**Reply by rahul-paigavan** [reply]
Understood, very helpful. Thanks.
